File: 118A - String Task/StringTask.scala
// http://codeforces.com/problemset/problem/118/A

import scala.io.StdIn.readLine

object StringTask {
    def main(args: Array[String]): Unit = {
        val vowels: Set[Char] = "aeiouy".toSet

        var line: String = readLine().toLowerCase()
        line = line.filterNot(vowels.contains(_))
        val dotted: String = line.mkString(".")
        println(f".${dotted}%s")
    }
}

File: 118A - String Task/StringTask.py
# http://codeforces.com/problemset/problem/118/A

vowels = "aoyeui"

line = input().lower()

no_vowels = line.translate({ord(c): None for c in vowels})
dotted = '.'.join(list(no_vowels))
print(f".{dotted}")